great_expectations/cli/suite.py

```diff
@@ -338,29 +338,38 @@ def _suite_new(suite: str, directory: str, empty: bool, jupyter: bool, view: boo
 
 
 @suite.command(name="delete")
-@click.option("--suite", "-es", default=None, help="Expectation suite name.")
+@click.argument("suite")
 @click.option(
     "--directory",
     "-d",
     default=None,
-    help="Delete expectation suite.",
+    help="Delete an expectation suite.",
 )
+@mark.cli_as_experimental
 def suite_delete(suite, directory):
-    """Delete an expectation suite from the expectation store."""
+    """
+    Delete an expectation suite from the expectation store.
+    """
+    usage_event = "cli.suite.delete"
     context = toolkit.load_data_context_with_error_handling(directory)
     suite_names = context.list_expectation_suite_names()
     if len(suite_names) == 0:
```

Comment:
```suggestion
    if not suite_names
```

to be more pythonic (I know it's not your change though 😅)